localstack/services/s3/v3/storage/core.py

```diff
@@ -181,6 +183,17 @@ def get_multipart(self, bucket: BucketName, upload_id: MultipartUploadId) -> S3S
     def remove_multipart(self, bucket: BucketName, s3_multipart: S3Multipart):
         pass
 
-    @abc.abstractmethod
+    def create_bucket(self, bucket: BucketName):
+        pass
+
+    def delete_bucket(self, bucket: BucketName):
+        pass
+
+    def flush(self):
+        pass
+
     def close(self):
         pass
+
+    def reset(self):
+        pass
```

Comment:
maybe some doc strings would be helpful to explain what the different behavior of flush,close,reset are

Reply:
Will do 👍 funnily, I don't think we use flush anywhere at the moment, as persistence is always "on" for the filesystem storage backend. But we could keep it for some fun behaviour with community pods or something?